### PR TITLE
add NEOK asset

### PR DESCRIPTION
**Pull request by sirnicolaz**
## Description

NEOK is the token of NEOkingdom DAO, legally recognized as  NEOKingdom DAO OÜ. Some information:
- [Estonian Business Registry](https://ariregister.rik.ee/eng/company/16638166/Neokingdom-DAO-O%C3%9C)
- [GitHub](https://github.com/NeokingdomDAO/contracts)
- [Discord](https://discord.gg/JFXEWmeDw6)
- [Website](https://www.neokingdom.org/)

Further information can be found in the [EVMOS governance proposal](https://app.evmos.org/governance?id=144) that allowed our token to become an IBC coin.

We make this PR in order to list our token, NEOK, on Osmosis Frontier and as an unverified asset on the Osmosis Frontend (after which we will go ahead with the governance proposal to verify it).

### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry). Please refer to the [merged PR](https://github.com/cosmos/chain-registry/pull/2731)
- [x] Add asset to bottom of `zone_assets.json`.

### On-chain liquidity

- [x] The submitting team will ensure a pool will be created soon. Please hold off until further communication, which will notify of the Pool ID. (Please set the PR's status to Draft until the pool has been created)

Pool IDS:
- neok/osmo pool id: 1121 (tx [F9914D903A4A758C1785FE31AC76E3164A98E0F99D50F674DC331B75A4AAB7AC](https://www.mintscan.io/osmosis/transactions/F9914D903A4A758C1785FE31AC76E3164A98E0F99D50F674DC331B75A4AAB7AC))
- neok/axlUSDC pool id: 1122 (tx [0BF2DE1BFCECDA766780CE639290F5C730852D6B9ED5938E1226D79413F62100](https://www.mintscan.io/osmosis/transactions/0BF2DE1BFCECDA766780CE639290F5C730852D6B9ED5938E1226D79413F62100))

### Validation Testing

If adding or updating a chain or asset, the changes must be validated from the updated build of the Osmosis Frontend. Testers will need a small amount of tokens to validate. 
  - [ ] Validaters can buy a small amount of this asset from Pool ID: 1121 for neok/osmos, 1122 for neok/axlUSDC
  - [ ] Preview build has been validated.

## Additional Details

Initial Liquidity for the NEOK Token would be provided in the NEOK/axlUSDC and NEOK/OSMO pairs. Liquidity will be seeded by one of the original stakeholders and investors of the project in the 50.000 USD range.

For now we have created 2 pools with 3 NEOK/3 OSMO and 3 NEOK/3 axlUSDC respectively, in order for you to test them. If more is needed, please let us know.

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -1630,6 +1630,13 @@
       "path": "transfer/channel-522/uqsomm",
       "osmosis_main": true,
       "osmosis_frontier": true
+    },
+    {
+      "chain_name": "evmos",
+      "base_denom": "erc20/0x655ecB57432CC1370f65e5dc2309588b71b473A9",
+      "path": "transfer/channel-204/erc20/0x655ecB57432CC1370f65e5dc2309588b71b473A9",
+      "osmosis_main": true,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:1638]: please revert osmosis_main to false for now

**Comment by JeremyParish69** [comment]
Any update on a transfer interface?

**Comment by sirnicolaz** [comment]
EVMOS added it to staging, so I believe it should be ready in a matter of days Best,Nicola Am 12.09.2023 um 22:03 schrieb JeremyParish69 ***@***.***>:﻿
Any update on a transfer interface?

**Comment by sirnicolaz** [comment]
@JeremyParish69 the NEOK asset has been integrated in the EVMOS app. Please find it here
https://app.evmos.org/assets

Does that fulfill the requirements?

**Comment by JeremyParish69** [comment]
@sirnicolaz Sorry, there have ben changes. Could I please ask that your resolve conflicts one more time and tag me? Now that it's merged in our frontend I can merge this right away.

**Comment by sirnicolaz** [comment]
@JeremyParish69 conflict resolved. I also switched to what I figured was the new flag: `osmosis_verified: false`

**Pull request merged**